pkg/kubecost/asset.go

```diff
@@ -3841,6 +3841,7 @@ func (asr *AssetSetRange) InsertRange(that *AssetSetRange) error {
 	}
 
 	var err error
+	var as *AssetSet
```

Comment:
Which makes this unnecessary.

Reply:
The use of `:=` inside the `for` loop creates a new `err` variable that does not exist outside the loop where `err` is being returned.

Reply:
You are correct. My apologies.

Reply:
https://go.dev/play/p/JHopQtN4dnY